ixmp/backend/jdbc.py

```diff
@@ -748,32 +749,31 @@ def start_jvm(jvmargs=None):
         .. _`JVM documentation`: https://docs.oracle.com/javase/7/docs
            /technotes/tools/windows/java.html)
     """
-    # TODO change the jvmargs default to [] instead of None
     if jpype.isJVMStarted():
         return
 
-    jvmargs = jvmargs or []
-
     # Arguments
-    args = [jpype.getDefaultJVMPath()]
-
-    # Add the ixmp root directory, ixmp.jar and bundled .jar and .dll files to
-    # the classpath
-    module_root = Path(__file__).parents[1]
-    jarfile = module_root / 'ixmp.jar'
-    module_jars = list(module_root.glob('lib/*'))
-    classpath = map(str, [module_root, jarfile] + list(module_jars))
-
-    sep = ';' if os.name == 'nt' else ':'
-    args.append('-Djava.class.path={}'.format(sep.join(classpath)))
-
-    # Add user args
-    args.extend(jvmargs if isinstance(jvmargs, list) else [jvmargs])
-
-    # For JPype 0.7 (raises a warning) and 0.8 (default is False).
-    # 'True' causes Java string objects to be converted automatically to Python
-    # str(), as expected by ixmp Python code.
-    kwargs = dict(convertStrings=True)
+    args = jvmargs if isinstance(jvmargs, list) else [jvmargs]
+
+    # Base for Java classpath entries
+    cp = Path(__file__).parents[1]
+
+    # Keyword arguments
+    kwargs = dict(
+        # Given 'lib/*' JPype will only glob '*.jar', so glob here explicitly
+        classpath=map(str, chain([cp / 'ixmp.jar'], cp.glob('lib/*'))),
+
+        # For JPype 0.7 (raises a warning) and 0.8 (default is False).
+        # 'True' causes Java string objects to be converted automatically to
+        # Python str(), as expected by ixmp Python code.
+        convertStrings=True,
+    )
+
+    if os.name == 'nt':
+        print('JAVA_HOME:', os.environ.get('JAVA_HOME', '(not set)'))
```

Comment:
is there any reason to use print instead of logger?

Reply:
Oh, I should remove that entirely…or change to log.debug(), yes. Sorry.